### PR TITLE
LPS-200907 Video streaming subtitles

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Now subtitles are visible if they are provided via config to the Fragment. 
This is done in a new module without test, so no other test must be affected. If ci:test:relevant doesn't pass I will force forward. 

**Steps to test it:**

1. Edit home page and add **Video Streaming** fragment
2. Configure it. **URL** https://s3-us-west-2.amazonaws.com/s.cdpn.io/122116/2010-0927-KMOV-News-4.mp4 and **subtitles** https://s3-us-west-2.amazonaws.com/s.cdpn.io/122116/2010-0518-KMOV-News-4_3.vtt 
3. Publish page. To see subtitles, you must select it from the control bar. 

Screenshot 2023-11-15 at 13 15 03

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-video-streaming/src/main/java/com/liferay/fragment/video/streaming/VideoStreamingFragmentRenderer.java [hunk 1]
```diff
@@ -137,6 +139,13 @@ public void render(
 						getConfiguration(fragmentRendererContext),
 						fragmentEntryLink.getEditableValues(),
 						fragmentRendererContext.getLocale(), "url")));
+			httpServletRequest.setAttribute(
+				VideoStreamingWebKeys.VIDEO_STREAMING_SUBTITLES,
+				GetterUtil.getString(
+					_fragmentEntryConfigurationParser.getFieldValue(
+						getConfiguration(fragmentRendererContext),
+						fragmentEntryLink.getEditableValues(),
+						fragmentRendererContext.getLocale(), "subtitles")));
```
Comment (marcogalluzzi) [inline modules/apps/fragment/fragment-video-streaming/src/main/java/com/liferay/fragment/video/streaming/VideoStreamingFragmentRenderer.java:148]: The "subtitles" label is missing in `Language.properties`

**Review by marcogalluzzi — CHANGES_REQUESTED** [review]
I suppose this is still work in progress. If you want to check now, the `autoplay` option is not working anymore.

**Comment by ambrinchaudhary** [comment]
ci:forwad:force

**Review by marcogalluzzi — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Pull request closed without merging**